extensions/applicationinsights-properties-js/src/Context/Session.ts

```diff
@@ -116,6 +105,20 @@ export class _SessionManager {
                 const session = _self.automaticSession;
                 _setStorage(session.id, session.acquisitionDate, session.renewalDate);
             };
+
+            function _storageNamePrefix(): string {
```

Comment:
I'm not a fan of this pattern, but I understand why :-(

I'm currently working on fixing the referencing issue, so I *might* undo these functions with my PR (after you have submitted this one)

Reply:
Thanks Nev! I added TODO in the code!